t/find2perl.t

```diff
@@ -52,11 +52,16 @@ my @test_files =
     (
         { name => "abc" },
         { name => "acc", mtime => time() - HOUR * 48 },
+        { name => "adc", atime => time() - HOUR * 2 },
         { name => "ac", content => "x" x 10 },
         { name => "somedir", type => "d" },
         { name => "link", type => "l", target => "abc" },
-        { name => "symlink", type => "s", target => "brokenlink" },
+
+# Comment out, since trying to create this symlink dies:
```

Comment:
This should definitely not be part of this commit. Probably it needs to be skipped conditionally based on operating system.

Reply:
Fix in https://github.com/Leont/app-find2perl/pull/4/commits/90f1937bbba5186056f9ebb327237c53273559b2